### PR TITLE
fix #36388 berrybase.de

### LINKED ISSUE #36388: berrybase.de
### Issue URL (Annoyance)
[https://www.berrybase.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.berrybase.de%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | v3.1.77η (1.4.78cl)
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Block third-party Authorization header, Self-destructing first-party cookies (4320), Self-destructing third-party cookies (180), Disable cache for third-party requests, Block WebRTC, Block Location API, Hide your Referrer from third-parties (), Hide your Referrer from third-parties, Hide your User-Agent (), Hide your IP address (209.141.53.254)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | AdGuard Mobile Ads,<br>AdGuard Annoyances,<br>AdGuard Base,<br>hufilter,<br>Malware Domains,<br>Korean Adblock List,<br>Fanboy's Anti-thirdparty Fonts,<br>NoCoin Filter List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/36388

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ acc.cv.ua###facebookLikeBox
 autotrader.com#$#.internationalOverlay { display: none !important; }
 autotrader.com#$#.modal-backdrop { display: none !important; }
 autotrader.com#$#body { overflow: visible!important; }
+berrybase.de#$#div[id^="tsbadgeResponsiveTop"] { position: absolute!important; left: -3000px!important; }
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/popups.txt]: Not found in code.

Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt]: <details><summary>Screenshot:</summary>

Снимок экрана 2019-07-08 в 18 31 06

</details><br/>

Comment (BlazDT) [reply AnnoyancesFilter/sections/popups.txt]: Not sure if we should hide that element as such banner can help the user to decide if he wants to order from there or not. Especially if that banner has a link to a trusted rating site.

Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt]: Please be careful, look on third-party resources. This is trustedshops.com widget, and can be blocked in Annoyances filter,

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**